### ISSUE TITLE
Reduce the amount of blocks you can find in treasure chests

### ISSUE THREAD

**Issue by LoneWolfHT** [MT-CTF/capturetheflag#1020]
Currently teams end up with so many blocks that they're never in need of any. Reducing the amount you can find may reduce the amount of pillaring you see.

**Comment by xenonca** [comment]
I doubt that. Those who pillar are usually not the same people who need blocks to build defensive structures/the base. I'm afraid we could end up with even more people complaining about the pillaring as they'll now miss blocks for their base building.

**Comment by farooqkz** [comment]
For that the amount of blocks placed per second could be further limited.

**Comment by mieu12** [comment]
Reducing the amount of blocks may not help with the auto clickers, but in my opinion there are way too many blocks in treasure chests which makes them seem worthless so then they are wasted.

**Comment by Calinou** [comment]
> For that the amount of blocks placed per second could be further limited.

Limiting the number of blocks placed per second may be too limiting when "panic-building" to block off an entrance. It would probably be better to limit on a rolling 10-second interval or something instead.

**Comment by LoneWolfHT** [comment]
> I doubt that. Those who pillar are usually not the same people who need blocks to build defensive structures/the base. I'm afraid we could end up with even more people complaining about the pillaring as they'll now miss blocks for their base building.

Going through 5 treasure chests I can fill the team chest 3/4 full with blocks. I think there's a decent excess

**Comment by farooqkz** [comment]
> > I doubt that. Those who pillar are usually not the same people who need blocks to build defensive structures/the base. I'm afraid we could end up with even more people complaining about the pillaring as they'll now miss blocks for their base building.
> 
> Going through 5 treasure chests I can fill the team chest 3/4 full with blocks. I think there's a decent excess

Well pros like `s****` really need these blocks :P

**Comment by magnetar47** [comment]
I agree with @xenonca that the people who pillar all over the map aren't typically the people who build bases.

Regarding the concern that there are too many blocks in the chests than what's required for building bases, I think we must also consider the fact that bridges are valid strategies, and sky-bridging from one base to the other can consume a whack-ton of blocks. While I personally do think that there may be a *slight* excess of blocks in these chests, we mustn't reduce the number so much that sky-bridging isn't viable at all.

I also feel that rate-limiting block placement isn't a good way to address excessive pillaring, as there are a hundred and one valid use-cases for rapid block placement (plis don't ask me to list them all 😅). Heck, even building bases requires rapid block placement, given such a short build time. If you think about it, building walls is literally pillaring at every point along a line, (depending on your approach :P).

**Comment by LoneWolfHT** [comment]
Maybe I'll start by reducing the block spawns in classic, which used to involve more mining for blocks (IIRC)

**Comment by farooqkz** [comment]
I think this can be closed as we don't have that much of pillaring anymore in 2025
